### ISSUE TITLE
SVGJS 2.3.2 hangs in chrome while rendering offscreen

### ISSUE THREAD

**Issue by ertong** [wout/svg.js#514]
```html
<script src="https://code.jquery.com/jquery-2.2.4.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/svg.js/2.3.2/svg.js"></script>
A
<div class="clock"></div>
B
<script>

    !function(){
        var container = $("<div></div>"); //1
        var draw = SVG(container[0]).size(100,100);

        draw = draw.viewbox(0,0, 100, 100);

        draw.text("afdsfd")
         .x(5)
         .y(5);
        $(".clock").append(container); //2
    }()
</script>
```

This code will make chrome browser unresponsive.
If we take version 2.3.1, everything will work.
If we place line 2 right after line 1, everything will work.

**Comment by Fuzzyma** [comment]
Simple answer: Dont render offscreen. Its not faster because svg.js has to work around this by placing the elements in the dom anyway to get its bbox (in this case `y()` will call bbox).

Actually 2.3.2 was released to fix exactly that bug. No idea why its there again. Have to look into that...

**Comment by Fuzzyma** [comment]
For some reason `SVG.Text` had its own `clone()` method which leaded to his error.
Fixed in 2.3.3
